lib/project_types/rails/commands/create.rb

```diff
@@ -10,13 +10,17 @@ class Base < ActiveResource::Base
         end
       USERAGENT
 
+      DEFAULT_RAILS_FLAGS = %w(--skip-spring)
+
       options do |parser, flags|
         # backwards compatibility allow 'title' for now
         parser.on('--title=TITLE') { |t| flags[:title] = t }
         parser.on('--name=NAME') { |t| flags[:title] = t }
         parser.on('--organization_id=ID') { |url| flags[:organization_id] = url }
         parser.on('--shop_domain=MYSHOPIFYDOMAIN') { |url| flags[:shop_domain] = url }
         parser.on('--type=APPTYPE') { |url| flags[:type] = url }
+        parser.on('--db=DB') { |db| flags[:db] = db }
+        parser.on('--rails_opts=RAILSOPTS') { |opts| flags[:rails_opts] = opts }
```

Comment:
That could be a good idea - do you have suggestions for how to do so?

Reply:
Disclaimer: I'm not very knowledgeable on ruby.

Can we run a similar parser on the contents of the options? So essentially do the same for the 'inner' params without necessarily forcing them.

Reply:
We shouldn't really need any sanitization because these are flags passed into the cli from the user, proxied to the rails new command for the user. Any other sanitation like escaping quotes and spaces should either be done by the shell or by the user so that the shell will pass the arguments along